### PR TITLE
Feature/SM-1 Implemented the login story

### PR DESCRIPTION
**Pull request by Pangolini20**

### CODE REVIEW

**Comment by Navzar99** [comment]
Please resolve conflicts!” and then click Comment.

File: src/loose/oose/fis/lab/student/manager/Controller.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package loose.oose.fis.lab.student.manager;
```
Comment (Navzar99) [inline src/loose/oose/fis/lab/student/manager/Controller.java]: Remove unused file

File: src/loose/oose/fis/lab/student/manager/Main.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package sample;
```
Comment (Navzar99) [inline src/loose/oose/fis/lab/student/manager/Main.java]: Remove unused file🤔

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            return;
+        }
+
+        if (username.equals("student") && password.equals("student")) {
+            loginMessage.setText("Logged in as student!");
+            return;
+        }
+
+        if (username.equals("teacher") && password.equals("teacher")) {
+            loginMessage.setText("Logged in as teacher!");
```
Comment (Navzar99) [inline src/loose/oose/fis/lab/student/manager/controllers/LoginController.java]: If login as a teacher succeeds, the user should be taken to the view-students scene”.

File: src/loose/oose/fis/lab/student/manager/Controller.java (hunk 1 above)
Comment (Pangolini20) [reply src/loose/oose/fis/lab/student/manager/Controller.java]: Problem solved !

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java (hunk 3 above)
Comment (Pangolini20) [reply src/loose/oose/fis/lab/student/manager/controllers/LoginController.java]: I finished it I'm sorry i forgot

**Pull request merged**